src/inausoft.netCLI/Commands/HelpCommandHandler.cs

```diff
@@ -45,23 +43,29 @@ public override int Run(HelpCommand command)
             }
             else
             {
-                var commandHandler = _commandHandlers.FirstOrDefault(it =>
-                    Attribute.IsDefined(it.GetCommandType(), typeof(CommandAttribute)) &&
-                    (Attribute.GetCustomAttribute(it.GetCommandType(), typeof(CommandAttribute)) as CommandAttribute).Name == command.SpecifiedCommandName);
+                var commandType = _configuration.CommandTypes.FirstOrDefault(it =>
+                    Attribute.IsDefined(it, typeof(CommandAttribute)) &&
+                    (Attribute.GetCustomAttribute(it, typeof(CommandAttribute)) as CommandAttribute).Name == command.SpecifiedCommandName);
 
-                if (commandHandler == null)
+                if (commandType == null)
                 {
                     throw new InvalidCommandException(command.SpecifiedCommandName, $"Command {command.SpecifiedCommandName} is invalid.");
                 }
 
-                StringBuilder message = new StringBuilder(command.SpecifiedCommandName);
+                StringBuilder message = new StringBuilder($"{command.SpecifiedCommandName}");
+
+                var properties = commandType.GetProperties().Where(it => Attribute.IsDefined(it, typeof(OptionAttribute)));
 
-                var properties = commandHandler.GetCommandType().GetProperties().Where(it => Attribute.IsDefined(it, typeof(OptionAttribute)));
+                if(properties != null)
+                {
+                    message.AppendLine(" [OPTIONS]");
+                    message.AppendLine("options:");
+                }
 
                 foreach(var property in properties)
                 {
-                    var option = (Attribute.GetCustomAttribute(property, typeof(OptionAttribute)) as OptionAttribute).Name;
-                    message.Append($" --{option} <{option.ToUpper()}>");
+                    var option = (Attribute.GetCustomAttribute(property, typeof(OptionAttribute)) as OptionAttribute);
+                    message.AppendLine($" --{option.Name} \t\t {option.HelpDescription}");
```

Comment:
Is it a purpose to use heavy `Stringbuilder` here? I would ask Jan about it ;)

Reply:
@jmarlew any insights?